locales/messages.de.json

```diff
@@ -1,35 +1,35 @@
-  "ErrorNoVSInstance": "Fehler: im Tripel {triplet}: Es wurde keine gültige Visual Studio-Instanz gefunden.",
+  "ErrorMissingVcpkgRoot": "Fehler: .vcpkg-root konnte nicht erkannt werden. Sollten Sie versuchen, eine selbst erstellte Kopie von vcpkg zu verwenden, muss die Umgebungsvariable VCPKG_ROOT definiert sein, um auf eine geklonte Kopie von {url} zu verweisen.",
+  "ErrorNoVSInstance": "Fehler: im Triplett {triplet}: Es wurde keine gültige Visual Studio-Instanz gefunden.",
   "ErrorNoVSInstanceAt": "     bei „{path}“",
-  "ErrorNoVSInstanceFullVersion": "    mit dem Präfix für die Toolsetversion {version}",
-  "ErrorNoVSInstanceVersion": "    mit der Toolsetversion {version}",
+  "ErrorNoVSInstanceFullVersion": "    mit dem Präfix für die Toolset-Version {version}",
+  "ErrorNoVSInstanceVersion": "    mit der Toolset-Version {version}",
   "ErrorRequireBaseline": "Fehler: Diese vcpkg-Instanz erfordert ein Manifest mit einer angegebenen Baseline, um mit Ports zu interagieren. Bitte fügen Sie dem Manifest „builtin-baseline“ hinzu, oder fügen Sie eine Datei „vcpkg-configuration.json“ hinzu, welche die Standardregistrierung neu definiert.\n",
   "ErrorRequirePackagesList": "Fehler: „vcpkg install“ erfordert eine Liste von Paketen für die Installation im klassischen Modus.",
   "ErrorRequirePackagesToInstall": "Fehler: Es wurden keine Pakete für die Installation aufgelistet, und es wurde kein Manifest gefunden.",
-  "ErrorVcvarsUnsupported": "Fehler: im Tripel {triplet}: Die Verwendung der Entwickleraufforderung von Visual Studio wird auf Nicht-Windows-Hosts nicht unterstützt.\nDefinieren Sie „VCPKG_CMAKE_SYSTEM_NAME“ oder „VCPKG_CHAINLOAD_TOOLCHAIN_FILE“ in der Tripeldatei.",
+  "ErrorVcvarsUnsupported": "Fehler: im Triplett {triplet}: Die Verwendung der Entwickleraufforderung von Visual Studio wird auf Nicht-Windows-Hosts nicht unterstützt.\nDefinieren Sie „VCPKG_CMAKE_SYSTEM_NAME“ oder „VCPKG_CHAINLOAD_TOOLCHAIN_FILE“ in der Tripelettdatei.",
   "ForceSystemBinariesOnWeirdPlatforms": "Die Umgebungsvariable VCPKG_FORCE_SYSTEM_BINARIES muss auf ARM-, S390x- und ppc64le-Plattformen festgelegt werden.",
   "FormattedParseMessageExpression": "    beim Ausdruck: {value}",
   "FormattedParseMessageLocation": "{path}:{row}:{column}: ",
   "GenerateMsgErrorParsingFormatArgs": "Fehler: Formatzeichenfolge für {value} wird analysiert:",
   "GenerateMsgIncorrectComment": "Die Nachricht {value} weist einen falschen Kommentar auf:",
   "GenerateMsgNoArgumentValue": "    {{{value}}} wurde in einem Kommentar angegeben, aber nicht in der Nachricht verwendet.",
   "GenerateMsgNoCommentValue": "    {{{value}}} wurde in der Nachricht verwendet, aber nicht kommentiert.",
-  "GraphCycleDetected": "Zyklus innerhalb des Diagramms erkannt bei {package_name}:",
```

Comment:
No it is not a diagram. It is a graph.

Reply:
why is this pluralized?

Reply:
(internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565236

Reply:
> why is this pluralized?

There is no plural in the German sentence?

Reply:
Graphen is pluralized no?

Reply:
Hm it's complicated. 
a graph => Ein Graph
multiple graphs => mehrere Graphen 
within a graph => in einem Graphen